.eslintrc.js

```diff
@@ -33,6 +33,15 @@ module.exports = {
 				step: false,
 			},
 		},
+		{
```

Comment:
Should this live in the `.eslintrc.js` of the FSE app?

Reply:
Yeah, I think that would make more sense :)

Reply:
Done!